app/src/main/cpp/ExternalVR.cpp

```diff
@@ -262,6 +262,7 @@ mozilla::gfx::VRControllerType GetVRControllerTypeByDevice(device::DeviceType aT
     case device::UnknownType:
     default:
       result = mozilla::gfx::VRControllerType::_empty;
+      assert(!"Unknown controller type.");
```

Comment:
What is that ! for?

Reply:
It's like ```assert (false)``` , kind of an alternative for ```assert_notreached``` , but it shows the "string" in the backtrace:

```
2023-09-29 16:51:46.120  3901-3983  libc                    com.igalia.wolvic.dev                A  /home/javi/devel/Wolvic/wolvic/app/src/main/cpp/ExternalVR.cpp:265: mozilla::gfx::VRControllerType crow::GetVRControllerTypeByDevice(device::DeviceType): assertion "!"Unknown controller type."" failed
```

Reply:
The assert added here raises SIGABRT when using Wolvic with `noapi`. Before that, I was able to start immersive sessions and use it for basic debugging with `noapi`, even without proper input controller was getting assigned on such configuration.